src/test/java/org/wof/foodministry/FoodEntryTestClass.java

```diff
@@ -43,13 +43,48 @@ public void testGetWeightFalse() {
         int result = entry.getWeight();
         assertNotEquals(10, result);
     }
-    //TODO: Create true and false test cases for the other FoodEntry class variables
-    //      Follow the pattern show for the getWeight() method
 
-    //TODO: Look into other unit test types to get complete testing coverage on your class
+    @Test
+    public void testGetWasteTrue() {
+        System.out.println("Testing getWaste() for true");
+        int result = entry.getWaste();
+        assertEquals(25, result);
+    }
 
     @Test
-    public void firstTest() {
-        Assert.assertTrue(true);
+    public void testGetWasteFalse() {
+        System.out.println("Testing getWaste() for false");
+        int result = entry.getWaste();
+        assertNotEquals(100, result);
+    }
+
+    @Test
+    public void testGetLocationTrue() {
+        System.out.println("Testing getLocation() for true");
+        String result = entry.getLocation();
+        assertEquals("Fairfax", result);
+    }
+
+    @Test
+    public void testGetLocationFalse() {
+        System.out.println("Testing getLocation() for False");
+        String result = entry.getLocation();
+        assertNotEquals("Clarendon", result);
+    }
     }
-}
+
```

Comment:
I don't see any test cases for `getDat()` or `setDate()`
Was this on purpose?
Check lines 16-21 of this file